### ISSUE TITLE
Set Postgres schema

### ISSUE THREAD

**Issue by EmileSpecs** [feathersjs-ecosystem/feathers-objection#104]
Can't seem to figure out how to set the Postgres schema.

In feathers-knex it can be set in die service options as well as through hooks (setting context.service.schema).

This doesn't seem to work with feathers-objection,  is there some way to set the schema (it obviously defaults to "public")?

**Comment by dekelev** [comment]
ObjectionJS has [models](https://github.com/feathersjs-ecosystem/feathers-objection#models) (JSON Schema format) that is used for data validation, hooks, describe relations and more.

ObjectionJS models won't help you with schema migration - for this you can use [Knex migrate tool](https://github.com/feathersjs-ecosystem/feathers-objection#db-migrations).

**Comment by EmileSpecs** [comment]
Hi @dekelev 
Thanks for the reply, I think you misunderstood my question.

Using **feathers-knex** my services are defined as:
```
module.exports = function (app) {
  const options = {
    Model: createModel(app),
    paginate: app.get('paginate'),
    **schema: app.get('postgres').connection.schema**
  };

  // Initialize our service with any options it requires
  app.use('/accounts', new Accounts(options, app));

  // Get our initialized service so that we can register hooks
  const service = app.service('accounts');

  service.hooks(hooks);
};
```
feathers-knex allows for a schema option in service options.

I cannot do the same with feathers-objection, it always defaults to the "public" schema.
I cannot initiate it in the knex connection (as you've shown above) as not all the services use the same schema.

Can I set the schema somewhere in a hook perhaps?

**Comment by dekelev** [comment]
Thanks @EmileSpecs , PR released with v5.6.0
